plugin-server/src/config/config.ts

```diff
@@ -170,6 +170,9 @@ export function getDefaultConfig(): PluginsServerConfig {
         SESSION_RECORDING_OVERFLOW_BUCKET_REPLENISH_RATE: 5_000_000, // 5MB/second uncompressed, sustained
         SESSION_RECORDING_OVERFLOW_BUCKET_CAPACITY: 200_000_000, // 200MB burst
         SESSION_RECORDING_OVERFLOW_MIN_PER_BATCH: 1_000_000, // All sessions consume at least 1MB/batch, to penalise poor batching
+
+        ENABLE_ERROR_EMBEDDING_TEAM_IDS: '',
```

Comment:
Do we want to roll this out to all teams at the end, or only teams that pay for it?
- If we plan to enable it for all teams but slowly ramp up the processing load, I recommend using the `_MAX_TEAM_ID` pattern like used by `POE_WRITES_ENABLED_MAX_TEAM_ID`: parse it as a string and just compare `event.team_id < ENABLE_ERROR_EMBEDDING_MAX_TEAM_ID`.
- If you want to have a specific list, you should reuse the `buildIntegerMatcher` utility, like we do for `SKIP_ELEMENTS_PARSING_PLUGINS` for example, to avoid several pitfalls we fell into in the past